src/main/java/kostka/moviecatalog/controller/AdministrationController.java

```diff
@@ -152,6 +152,32 @@ public String deleteUser(final @PathVariable Long userId) {
         return REDIRECT_ADMIN_VIEW;
     }
 
+    @GetMapping("/users/ban/{userId}")
+    public String banUser(final @PathVariable Long userId) {
+        LOGGER.info("ban user with id '{}' request", userId);
+
+        try {
+            userService.banUser(userId, true);
+        } catch (Exception e) {
+            LOGGER.error("User with id '{}' cannot be banned.", userId, e);
+        }
+        rabbitMqSender.sendRefreshAdminRequestToQueue();
+        return REDIRECT_ADMIN_VIEW;
+    }
+
+    @GetMapping("/users/unban/{userId}")
+    public String unbanUser(final @PathVariable Long userId) {
+        LOGGER.info("ban user with id '{}' request", userId);
+
+        try {
+            userService.banUser(userId, false);
```

Comment:
tip: klidně bych i v té service vytvořil 2 metody: ban, unban. Místo druhého bool parametru. Protože kdyby tohle nebylo volané z metody, která se jmenuje unbanUser, nemusí to být úplně jasné.

Obecně je pak čitelnější mít přesněj pojmenované metody, místo parametrů, které fungujou jako diskriminátor chování metody.

Reply:
diky za tip :) chtěl jsem se vyhnout duplicitě v kodu :)

Reply:
Klidně si tam tu metodu nech jako privátní, co bude brát ten bool, ale jako veřejnou, pro klienty toho API můžeš udělat ban/unban, které to na tu privání budou delegovat. Pak by si tam tu duplicitu mít nemusel :)

Reply:
krása :) díky